daphne/src/testing.rs

```diff
@@ -272,3 +357,103 @@ impl DapLeader<BearerToken> for MockAggregator {
         unreachable!("not implemented");
     }
 }
+
+// Information associated to a certain report for a given task and nonce to decide which bucket it would be put into.
```

Comment:
Docucomments begin with "///" rather than "//".

```suggestion
/// Information associated to a certain report for a given task and nonce to decide which bucket it would be put into.
```

Reply:
Fixed!